integration-tests/src/test/resources/domain-home-on-pv/create-domain.py

```diff
@@ -48,26 +48,158 @@ def getEnvVar(var):
 # Configure the Administration Server
 # ===================================
 cd('/Servers/AdminServer')
-# Dont set listenaddress, introspector overrides automatically with sit-config
-#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
 set('ListenPort', admin_port)
 set('Name', admin_server_name)
 
 create('T3Channel', 'NetworkAccessPoint')
 cd('/Servers/%s/NetworkAccessPoints/T3Channel' % admin_server_name)
 set('PublicPort', t3_channel_port)
-set('PublicAddress', 'junkvalue')
-# Dont set listenaddress, introspector overrides automatically with sit-config
-#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+set('PublicAddress', t3_public_address)
+set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
```

Comment:
We should not be setting listen-addresses to their exact expected value, as automatic situational config is always generated and always expected to override the value, and we need to verify that the automatic situational config is working.   A public-address, on the other hand, requires 'manually' setting to the correct value if it's different than the listen-address - this can be done in the original configuration or via 'custom overrides'.

Reply:
The same comment applies all listen-addresses in this change.

Reply:
Update create-domain.py with the one from samples which doesn't set listen-address.